ci/keystone/keystone.sh

```diff
@@ -60,7 +63,7 @@ case "$1" in
     pid=$!
     # Doing this after launching keystone will give it plenty of time to get
     # started without adding any wasteful calls to sleep:
-    pip install python-openstackclient
+    pip install python-openstackclient==3.2.0
```

Comment:
No reasons for that specific version, but since we were freezing keystone to a version while openstackclient would always be the latest, we would probably be bound to maybe hit sooner or later another version incompatibility in dependent libraries. Either way, I removed it from the change since it's not required to fix the gate.